tools/clang/lib/SPIRV/EmitSPIRVAction.cpp

```diff
@@ -1794,6 +1797,19 @@ case BO_##kind : {                                                             \
     return 0;
   }
 
+  /// \brief Returns the breakStack.
```

Comment:
Are these methods still necessary? In the `SPIRVEmitter` class, we have access to all the variables anyway.